contracts/programs/ocr2/src/lib.rs

```diff
@@ -789,7 +789,8 @@ fn calculate_reimbursement(juels_per_lamport: u64, _signature_count: usize) -> R
     let lamports_per_signature = fees.fee_calculator.lamports_per_signature;
     let lamports = lamports_per_signature * SIGNERS;
     let juels = lamports * juels_per_lamport;
-    Ok(juels)
+    let gjuels = juels / 1000000000; // return value as gjuels
```

Comment:
Should we be concerned about overflows in this code?

Reply:
Division can't overflow but it will panic if `juels == 0`. Can use `.checked_div(GIGA).ok_or(ErrorCode::Overflow)?`

Reply:
`let juels = lamports * juels_per_lamport;` could use a check, but we enable program-wide overflow checks anyway which will abort: https://github.com/smartcontractkit/chainlink-solana/blob/3813f358930e37874f2b6a0bce14e0ddeec0e8df/contracts/Cargo.toml#L11

Reply:
also will division ever overflow? (assuming they are u128 values) i can't figure out how that would be possible